data/sql/updates/pending_db_world/repandspawn.sql

```diff
@@ -0,0 +1,4 @@
+--lots of Reputation changes
```

Comment:
```suggestion
-- lots of Reputation changes
```

Reply:
now the purpose of spacing after -- is for the ci runs. --XX and -- XX is viewed differently with the workflow.